jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PushBlobStep.java

```diff
@@ -96,7 +85,13 @@ public BlobDescriptor call() throws IOException, RegistryException, ExecutionExc
         return blobDescriptor;
       }
 
-      registryClient.pushBlob(blobDescriptor.getDigest(), blob, null);
+      // if pushing to same registry then can use `mount`/`from` to try to optimize
```

Comment:
possible rewording:

If base and target images are in the same registry, then use `mount`/`from` to try mounting the BLOb from the base image repository to the target image repository and possibly avoid having to push the BLOb. See `https://docs.docker.com/registry/spec/api/#cross-repository-blob-mount` for details.